x-pack/plugins/security_solution/server/endpoint/services/actions/clients/lib/base_response_actions_client.ts

```diff
@@ -709,6 +722,53 @@ export abstract class ResponseActionsClientImpl implements ResponseActionsClient
     });
   }
 
+  protected sendActionCreationTelemetry(actionRequest: LogsEndpointAction): void {
+    const isAutomated =
+      (actionRequest.EndpointActions.data.alert_id &&
+        actionRequest.EndpointActions.data.alert_id.length > 0) ??
+      false;
```

Comment:
Can you use `this.options.isAutomated` instead here.

Reply:
done 25dbc39c400f0a7fcb436b04f43a1862ae740c89